site/src/job_queue/mod.rs

```diff
@@ -234,6 +234,30 @@ pub async fn enqueue_benchmark_request(
 
     // Target x benchmark_set x backend x profile -> BenchmarkJob
     for target in Target::all() {
+        // enqueue Runtime job and Rustc job, for each backend
+        for &backend in backends.iter() {
```

Comment:
Let's just do rustc only for LLVM + x64 for now, and runtime benchmarks only for LLVM. We don't support multiple codegen backends for either anyway at this moment.

Reply:
e335e3657f3c881f93c31d065fd1e13cc9046869 :+1: